### PR TITLE
Update rubygem-foreman_kernel_care to 1.2.0

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

File: packages/katello/rubygem-foreman_kernel_care/rubygem-foreman_kernel_care.spec [hunk 1]
```diff
@@ -1,74 +1,50 @@
...
 URL: https://github.com/maccelf/foreman_kernel_care
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(foreman_remote_execution) >= 1.5.6
-Requires: %{?scl_prefix}rubygem(katello) >= 3.8.0
```
Comment (nadjaheitmann) [inline packages/katello/rubygem-foreman_kernel_care/rubygem-foreman_kernel_care.spec:25]: I am not sure if it is correct that all the dependencies are gone from the spec file.

Comment (evgeni) [reply packages/katello/rubygem-foreman_kernel_care/rubygem-foreman_kernel_care.spec:25]: it is, they are now autogenerated

**Comment by evgeni** [comment]
I did merge https://github.com/theforeman/foreman-packaging/pull/9848 before seeing your PR Nadja, sorry!

**Comment by evgeni** [comment]
Actually, yours is better, let me rebase it.

**Comment by evgeni** [comment]
Couldn't push to your branch, so did it here: https://github.com/theforeman/foreman-packaging/pull/9849

Thanks again!

**Comment by nadjaheitmann** [comment]
Note to myself: stop opening packaging PRs ;)

**Pull request closed without merging**